doc/get-started.md

```diff
@@ -181,23 +181,26 @@ information on installing Reader SDK for iOS, see the
       with the photos stored on your mobile device.
 1. Update the `application:didFinishLaunchingWithOptions:` method in your app
```

Comment:
Please update the method name to the new swift method

Reply:
- [ ] update the method name to swift method name.

Reply:
Done.